tool/jt.rb

```diff
@@ -529,6 +529,13 @@ def build(*options)
   def build_truffleruby(*options, sforceimports: true)
     mx 'sforceimports' if sforceimports
 
+    env_path = "#{TRUFFLERUBY_DIR}/mx.truffleruby/env"
+    env_content = File.exist?(env_path) ? File.read(env_path) : ''
+    dynamic_import = 'DEFAULT_DYNAMIC_IMPORTS=/tools'
+    unless env_content.include? dynamic_import
```

Comment:
This should be:
```ruby
unless env_content.include? 'DEFAULT_DYNAMIC_IMPORTS'
```
as there could be a different value for DEFAULT_DYNAMIC_IMPORTS.